engine/src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -442,33 +447,28 @@ abstract class CromwellTestkitSpec extends TestKit(new CromwellTestkitSpec.TestW
     }
   }
 
-  def getWorkflowMetadata(workflowId: WorkflowId, key: Option[String] = None)(implicit ec: ExecutionContext): JsObject = {
+  def getWorkflowMetadata(workflowId: WorkflowId, key: Option[String] = None, serviceRegistryActor: ActorRef)(implicit ec: ExecutionContext): JsObject = {
```

Comment:
I thought you couldn't have a default in the middle of a parameter chain? Is this why you've needed to make the None explicit below?